positive_numbers.py

```diff
@@ -0,0 +1,7 @@
+# returns a list of the positive numbers the user has entered
+def get_positive_numbers():
+    numbers = input("Please enter a sequence of numbers seperated by comma:")
+    return list(filter(lambda number: int(number) > 0, numbers.split(",")))
```

Comment:
Nice use of filter, but notice that you were asked to return a list of type int.
Fix this.